cluster-scope/overlays/nerc-ocp-infra/kustomization.yaml

```diff
@@ -3,5 +3,5 @@ kind: Kustomization
 resources:
 - ../common
 - ../../base/operators.coreos.com/subscriptions/openshift-gitops-operator
```

Comment:
the subscription is part of the bundle

Reply:
Argh, that was a bad rebase :). This is why I like reviews :smile: